moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -160,6 +173,25 @@ class IKFastKinematicsPlugin : public kinematics::KinematicsBase
   std::vector<std::string> link_names_;
   const size_t num_joints_;
   std::vector<int> free_params_;
+  robot_state::RobotStatePtr robot_state_;
+
+  // The ikfast and base frame are the start and end of the kinematic chain for which the
+  // IKFast analytic solution was generated.
+  const std::string ikfast_tip_frame_ = "_EEF_LINK_";
+  const std::string ikfast_base_frame_ = "_BASE_LINK_";
+
+  // The transform tip and base bool are set to true if this solver is used with a kinematic
+  // chain that extends beyond the ikfast tip and base frame. The solution will be valid so
+  // long as there are no active, passive, or mimic joints between either the ikfast_tip_frame
+  // and the tip_frame of the group or the ikfast_base_frame and the base_frame for the group.
+  bool transform_tip_;
+  bool transform_base_;
```

Comment:
for bools, make clearer what they do:

``transform_tip_`` --> ``transform_tip_required_``
...